DESCRIPTION.md

```diff
@@ -0,0 +1,61 @@
+LangKit is an open-source text metrics toolkit for monitoring language models. It offers an array of methods for extracting relevant signals from the input and/or output text, which are compatible with the open-source data logging library [whylogs](https://whylogs.readthedocs.io/en/latest).
+
+## Table of Contents 📖
+
+- [Motivation](#motivation-)
+- [Features](#features-)
+- [Installation](#installation-)
+- [Usage](#usage-)
+- [Modules](#modules-)
+
+## Motivation 🎯
+
+Productionizing language models, including LLMs, comes with a range of risks due to the infinite amount of input combinations, which can elicit an infinite amount of outputs. The unstructured nature of text poses a challenge in the ML observability space - a challenge worth solving, since the lack of visibility on the model's behavior can have serious consequences.
```

Comment:
Is productionizing a word? 🤣

Reply:
This is not my content, but I did cleanup some of it with Andre. I'm pulling in from the beta/examples branch.